src/main/resources/requests/accountManagementRequests.ts

```diff
@@ -1,8 +1,12 @@
-import { accountManagementApi } from './utils/axiosConfig';
+import {accountManagementApi, getAccountManagementCredentials} from './utils/axiosConfig';
 import { Logger } from '@hmcts/nodejs-logging';
 
+const superagent = require('superagent');
 const logger = Logger.getLogger('requests');
+
 export class AccountManagementRequests {
+  public accountManagementUrl = process.env.ACCOUNT_MANAGEMENT_URL || 'https://pip-account-management.staging.platform.hmcts.net/';
+
```

Comment:
Why can't we use accountManagementApi from axiosConfig? Is there a special reason of creating accountManagementUrl?

Reply:
The accountManagementApi in axiosConfig is creating a new instance of axios with the config we have set I believe and the accountManagementUrl I have added is for use with Superagent, there is another example of where we've done this in dataManagementRequests.ts. If you would prefer I refactor this and grab the consts from axiosConfig instead  then can do :)

Reply:
No, that's fine. You can user Superagent. Api staging url is hardcoded in both accoutManagementRequest.ts and axiosConfig file as well. We should keep this hardcoded url in one place only. Can you get this URL from axiosConfig file? Something like:

In axiosConfig file:
export const accountManagementApiUrl = "https://pip-account-management.staging.platform.hmcts.net/";

and use this const in accountManagementRequest file.

Reply:
Done 👍